common/logger/src/struct_log.rs

```diff
@@ -179,20 +180,30 @@ pub fn struct_logger_set() -> bool {
 
 /// Initializes struct logger from STRUCT_LOG_FILE env var
 /// Can only be called once
-pub fn init_struct_log_from_env() -> Result<(), InitFileLoggerError> {
+pub fn init_struct_log_from_env() -> Result<(), InitLoggerError> {
     if let Ok(file) = env::var("STRUCT_LOG_FILE") {
         init_file_struct_log(file)
+    } else if let Ok(udp_address) = env::var("STRUCT_LOG_UDP_ADDR") {
```

Comment:
It could make more sense to have one env var(STRUCT_LOG_FILE) with different prefixes.
Smth like if starts_with("udp://") -> start udp logger, else -> start file logger

Multiple loggers are not allowed anyway for structure log, so there is no usecase for setting both env